packages/message-pickup-repository-dynamodb/tests/client.test.ts

```diff
@@ -0,0 +1,171 @@
+import { EncryptedMessage } from '@credo-ts/core'
+import { expect, suite, test } from 'vitest'
+import { DynamodbClientRepository } from '../src/client'
+
+const connectionId = '4ffdd113-117b-4827-9af5-28aa73ec4bad'
+const recipientDids = ['did:key:123', 'did:jwk:123', 'did:peer:3abba']
+const encryptedMessage: EncryptedMessage = {
+  ciphertext: 'ciphertext',
+  iv: 'iv',
+  protected: 'protected',
+  tag: 'tag',
+}
+
+suite('dynamodb client', () => {
+  let client: DynamodbClientRepository
+
+  test('initialize', async () => {
+    client = await DynamodbClientRepository.initialize({
```

Comment:
this breaks running a single test. I think it's better to do it as a `beforeAll`